cpp/src/examples/test_client.cc

```diff
@@ -90,98 +97,92 @@ void proxy_get_version(const string &host, const string &port,
 }
 
 using namespace std::chrono;
+void _bench_one_client(std::unique_ptr<Proxy_client> client,
+                       std::shared_ptr<alba::statistics::Statistics> stats_p,
+                       const string &namespace_, const string &object_name,
+                       const string &file_name, const int n) {
+
+    try{
+        const alba::Checksum *checksum = nullptr;
+        alba::statistics::Statistics &stats = *stats_p;
```

Comment:
statistics are tracked with this object, but it's never used to print/produce output?